### PR TITLE
feat: requiring viewpoint id on create viewpoint

### PR DESCRIPTION
**Pull request by jtblack-aws**
create_viewpoint now requires a viewpoint ID as part of the request instead of the tile server generating an arbitrary one.  This will better support data traceability through OSML tools and other integrated services.

Also removing old integ tests since the capability has migrated to the osml-tile-server-test package.

**Issue #, if available:** n/a

### Checklist

Before you submit a pull request, please make sure you have the following:
- [x] Code changes are compact and well-structured to facilitate easy review
- [x] Changes are documented in the README.md and other relevant documentation pages
- [x] PR title and description accurately reflect the changes and are detailed enough for historical tracking
- [x] PR contains tests that cover all new code and the code has been manual tested
- [x] All new dependencies are declared (if any), and no unnecessary libraries are added
- [x] Performance impacts (if any) of the changes are evaluated and documented
- [x] Security implications of the changes (if any) are reviewed and addressed
- [x] I have read the [Contributing Guidelines](../CONTRIBUTING.md) and agree to follow the [Code of Conduct](../CODE_OF_CONDUCT.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by edparris** [comment]
How is viewpoint_id different from viewpoint_name? 

It seems like we can/should combine those two fields and add some restrictions on length and allowed characters since the values are used in a URL.

Has the team talked through what happens if a ID/name is reused by a customer?

File: setup.cfg [hunk 1]
```diff
@@ -37,7 +37,7 @@ install_requires =
     geojson>=3.0.0
     pydantic>=2.0.0
     python-json-logger>=2.0.0
-    osml-imagery-toolkit>=1.4.0rc1
+    osml-imagery-toolkit>=1.4.0rc2
```
Comment (RanbirAulakh) [inline setup.cfg:40]: I believe imagery-toolkit released rc3 recently.

**Review by RanbirAulakh — APPROVED** [review]
We should remove load-test folder as well since those are already in `osml-tile-server-test`

File: setup.cfg (hunk 1 above)
Comment (jtblack-aws) [reply setup.cfg:40]: >= should take care of rc3 but I'll bump it again next time I am working on it.

**Pull request merged**